lib/managers/collaboration-whitelists.js

```diff
@@ -0,0 +1,225 @@
+ * @param {string} [options.marker] - Paging marker, retrieve records starting at this position in the list. Left blank to start at the beginning.
+ * @param {Function} [callback] - Passed a list of collaboration whitelists if successful, error otherwise
+ * @returns {Promise<Object>} A promise resolving to the collection of collaboration whitelists
+ */
+CollaborationWhitelists.prototype.getAll = function(options, callback) {
+
+	var params = {
+		qs: options
+	};
+
+	var apiPath = urlPath(BASE_PATH);
+	return this.client.wrapWithDefaultHandler(this.client.get)(apiPath, params, callback);
+};
+
+/**
+ * Delete a given collaboration whitelist.
+ *
+ * API Endpoint: '/collaboration_whitelist_entries/:collaborationWhitelistID'
+ * Method: DELETE
+ *
+ * @param {string} collaborationWhitelistID - Box ID of the collaboration whitelist being requested
+ * @param {Function} [callback] - Empty response body passed if successful.
+ * @returns {Promise<void>} A promise resolving to nothing
+ */
+CollaborationWhitelists.prototype.delete = function(collaborationWhitelistID, callback) {
+
+	var apiPath = urlPath(BASE_PATH, collaborationWhitelistID);
+	return this.client.wrapWithDefaultHandler(this.client.del)(apiPath, null, callback);
+};
+
+/**
+ * Adds a Box User to the collaboration whitelist.
+ *
+ * API Endpoint: '/collaboration_whitelist_exempt_targets'
+ * Method: GET
+ *
+ * @param {string} userID - The ID of the Box User to be added to the whitelist
+ * @param {Function} callback - Passed a collaboration whitelist for user if successful, error otherwise
+ * @returns {Promise<Object>} A promise resolving to a user collaboration whitelist
+ */
+CollaborationWhitelists.prototype.addUser = function(userID, callback) {
+
+	var params = {
+		body: {
+			user: {
+				id: userID,
+				type: 'user'
+			}
+		}
+	};
+
+	var apiPath = urlPath(TARGET_ENTRY_PATH);
+	return this.client.wrapWithDefaultHandler(this.client.post)(apiPath, params, callback);
+};
+
+/**
+ * Retrieves information about a collaboration whitelist for user by whitelist ID.
+ *
+ * API Endpoint: '/collaboration_whitelist_exempt_targets/:userWhitelistID'
+ * Method: GET
+ *
+ * @param {string} userWhitelistID - The ID of the collaboration whitelist
+ * @param {Object} [qs] - Additional options can be passed with the request via querystring. Can be left null in most cases.
+ * @param {Function} callback - Passed the collaboration whitelist information for a user if it was acquired successfully
+ * @returns {Promise<Object>} A promise resolving to the collaboration whitelist object
+ */
+CollaborationWhitelists.prototype.getForUser = function(userWhitelistID, qs, callback) {
+
+	var params = {
+		qs: qs
+	};
+
+	var apiPath = urlPath(TARGET_ENTRY_PATH, userWhitelistID);
+	return this.client.wrapWithDefaultHandler(this.client.get)(apiPath, params, callback);
+};
+
+/**
+ * Retrieves information for all user collaboration whitelists.
```

Comment:
These should be updated with language that matches the API docs a bit more closely — something like "Retrieve a list of all exemptions to an enterprise's collaboration whitelist"?